service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/diagnosticreport/ObservationMapper.java

```diff
@@ -120,7 +127,51 @@ public String map() {
             );
         }
 
-        private String prepareNarrativeStatements(Observation observation) {
+        private Optional<String> prepareObservationStatement(Observation observation, CompoundStatementClassCode classCode) {
```

Comment:
GitHub diff may look a bit strange here, only because I've moved some methods up the class for readability. `prepareObservationStatement` was moved up the class, and now returns `<Optional> String` rather than `String`